src/Test/Utilities/Portable/CommonTestBase.cs

```diff
@@ -116,19 +123,19 @@ public abstract partial class CommonTestBase : TestBase
             int? expectedReturnCode = null,
             string[] args = null,
             EmitOptions emitOptions = null,
-            bool verify = true)
+            Verification verify = Verification.Passes)
         {
             Assert.NotNull(compilation);
 
             Assert.True(expectedOutput == null ||
                 (compilation.Options.OutputKind == OutputKind.ConsoleApplication || compilation.Options.OutputKind == OutputKind.WindowsApplication),
                 "Compilation must be executable if output is expected.");
 
-            if (verify)
+            if (verify == Verification.Passes)
             {
                 // Unsafe code might not verify, so don't try.
                 var csharpOptions = compilation.Options as CSharp.CSharpCompilationOptions;
-                verify = (csharpOptions == null || !csharpOptions.AllowUnsafe);
+                verify = (csharpOptions == null || !csharpOptions.AllowUnsafe) ? Verification.Passes : Verification.Skipped;
```

Comment:
:memo: We have a follow-up issue to tighten this as well (we shouldn't assume that all tests with unsafe code can't be verified).
[https://github.com/dotnet/roslyn/issues/23130](https://github.com/dotnet/roslyn/issues/23130)